lib/portlayer/exec/container.go

```diff
@@ -174,9 +175,6 @@ func (c *Container) Commit(ctx context.Context, sess *session.Session, h *Handle
 		c.vm = vm.NewVirtualMachine(ctx, sess, res.Result.(types.ManagedObjectReference))
 		c.State = StateCreated
 
-		// align the handle state w/the container
```

Comment:
I introduced the "Creating" state so it wasn't needed anymore -- previously when you created a containerVM the state was "Stopped" thus causing the issue.